### DISCUSSION TITLE
Saving the assembled file

### DISCUSSION THREAD

**Discussion by Aurelio-AVB** [mortbopet/Ripes#211] [Q&A] [answered]
Hi, 

When I am trying to save the assembled file using save as from assembly source code it generates a empty .bin file. Is there anything I have to setup in order to get the correct file
<img width="500" alt="save_as" src="https://user-images.githubusercontent.com/61214409/174202019-af8c6da6-514a-4b51-bb78-054d6c2652fb.png">

**Comment by edgardogho** [comment]
Can reproduce the issue on linux.

**Comment by mortbopet** [comment, marked as answer]
apologies for the wait - looks like there was a stray negation sign that resulted in this behavior. Fixed in https://github.com/mortbopet/Ripes/commit/4992a2de3e261a9d27abf112c72ea519a6dba2fe.
